src/com/superiornetworks/icarus/commands/Command_god.java

```diff
@@ -28,11 +28,10 @@ public boolean run(CommandSender sender, Command cmd, String label, String[] arg
                 return true;
             }
             Player player = (Player) sender;
-            if (!ICM_SqlHandler.hasDoomHammer(player.getName()))
```

Comment:
You have removed this (Great) but not replaced it with what should be there, a check for if they have god mode.